tests/ProjectsTest.php

```diff
@@ -51,7 +51,7 @@ private function initTestProject($organizationId)
         $this->assertArrayHasKey('limits', $foundProject);
         $this->assertTrue(count($foundProject['limits']) > 1);
         $this->assertArrayHasKey('metrics', $foundProject);
-        $this->assertEquals('mysql', $foundProject['defaultBackend']);
+        $this->assertEquals('snowflake', $foundProject['defaultBackend']);
```

Comment:
@pivnicek how did you make this working? It'll required also changes in default templates. But we are also removing mysql backend we'll have to do this changes so I'll prepare it

Reply:
ah, yes, I changed the default backend locally to snowflake in my template.